numpy/lib/function_base.py

```diff
@@ -764,6 +764,22 @@ def copy(a, order='K', subok=False):
     >>> x[0] == z[0]
     False
 
+    Note that ``np.copy`` will not copy the elements of object
```

Comment:
```suggestion
    Note that ``np.copy`` will not copy object elements within
```
Then this also covers structured arrays with object fields.

Reply:
If we want to get the shallow phrase in there can we change this to:
```
Note that ``np.copy`` is a shallow copy and will not copy object elements within
```
@eric-wieser ? otherwise this looks good to me.